src/Polly.Core/CircuitBreaker/Health/HealthMetrics.cs

```diff
@@ -0,0 +1,26 @@
+namespace Polly.CircuitBreaker.Health;
+
+internal abstract class HealthMetrics
+{
+    private const short NumberOfWindows = 10;
+    private static readonly TimeSpan ResolutionOfCircuitTimer = TimeSpan.FromMilliseconds(20);
```

Comment:
Just curious as to why 20ms?

Reply:
Dunno, this is the value that Polly V7 used. I kept it unchanged.

Reply:
"Same as before" is a good enough reason 😄